data/abilities.ts

```diff
@@ -3799,12 +3799,12 @@ export const Abilities: {[abilityid: string]: AbilityData} = {
 		num: 28,
 	},
 	tangledfeet: {
-		onModifyAccuracyPriority: 6,
+		onModifyAccuracyPriority: -1,
 		onModifyAccuracy(accuracy, target) {
 			if (typeof accuracy !== 'number') return;
 			if (target?.volatiles['confusion']) {
 				this.debug('Tangled Feet - decreasing accuracy');
-				return accuracy * 0.5;
+				return this.chainModify(0.5);
```

Comment:
I think chainModify automatically turns this into a 4096, but it may be better for consistency to use 0x800, 0x1000.

Reply:
I checked other places e.g. Defeatist and they all seem to use the readable value if it has an exact binary representation.

Reply:
As long as it actually uses 2048 when chain modifying, it's ok.